pkg/virt-api/webhooks/s390x.go

```diff
@@ -0,0 +1,64 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+/*
+ * s390x utilities are in the webhooks package because they are used both
+ * by validation and mutation webhooks.
+ */
+package webhooks
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+const (
+	defaultCPUModelS390x = v1.CPUModeHostPassthrough
```

Comment:
@jschintag What is the reason for choosing hte host pass-thru model instead of the default host-model?

Reply:
> @jschintag What is the reason for choosing hte host pass-thru model instead of the default host-model?

Mainly because i oriented myself here on arm64, which does the same. I guess it is in this instance more a matter of preference. Though `DefaultCPUModel` (currently `CPUModeHostModel`) might be a better option here to enable livemigration.

Reply:
That's basically the background I liked to understand. So my preference would be use the `DefaultCPUModel` then. 

@alicefr @andreabolognani @jschintag Wdyt?

Reply:
No particular opinion one way or another.

On aarch64 `host-passthrough` is pretty much a forced choice right now, as migration between machines is largely impossible and named CPU models for KVM are not a thing.

If the situation on s390x is such that it is reasonably possible to use `host-model` and obtain a migratable CPU model as a result, the same way it is on x86_64, then I'd say go for it.

Reply:
I have changed the implementation to use the current default implementation (what x86 is doing). This either takes whatever is configured in the cluster, or falls back to the default, which is `host-model`.

Reply:
After further looking into the implementation and the associated test `pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go`(L188+) i changed the implementation a bit, moving the call of `setDefaultCPUModel` out of the arch specific section, to ensure the CPU model is set should the arch specific part not do it. Additionally in the test i changed the `if arch == ""` part to use the default value as default and only change it for arm64, since that is the only arch that differs here.